### ISSUE TITLE
Delay on send message

### ISSUE THREAD

**Issue by allocenx** [humhub/humhub-modules-mail#60]
As title says, when i click on send button there is a short delay but noticeable and than redirects to the mail page, why it does that? Its a bug does not it?

**Comment by feldrim** [comment]
It could be done without reloading page I guess. Maybe some ajax stuff could do that. At least it would look cool while waiting on that delay period.

**Comment by buddh4** [comment]
Duplicate of https://github.com/humhub/humhub-modules-mail/issues/16
